src/Common/Commands.Common/PowerBICmdlet.cs

```diff
@@ -129,6 +129,14 @@ protected void AssertProfileExists()
             }
         }
 
+        // This is the default value for ParameterSetName
+        private string parameterSet = string.Empty;
+        public string ParameterSet
+        {
+            get => !string.IsNullOrEmpty(this.ParameterSetName) ? this.ParameterSetName : this.parameterSet;
```

Comment:
Two options, default parameterSet by doing:
private string parameterSet = this.ParameterSetName

Or in get do: this.parameterSet ?? this.ParameterSetName